sample-apps/spark/src/main/java/com/amazon/sampleapp/MetricEmitter.java

```diff
@@ -0,0 +1,75 @@
+package com.amazon.sampleapp;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.common.Labels;
+import io.opentelemetry.metrics.LongCounter;
+import io.opentelemetry.metrics.LongValueRecorder;
+import io.opentelemetry.metrics.Meter;
+
+public class MetricEmitter {
```

Comment:
What's the point of `MetricEmitter`? It seems to be the same in both sample apps, so it's not actually related to a framework

Reply:
MetricEmitter is related to a sample app. We are performing validation on Metric Collected by SDK and sample app only emits metric on `/outgoing-http-call` path. MetricEmitter is required for every sample apps.

Reply:
I think I get it - but it is copy-paste code among all the apps. I guess it'd be good if we can publish a testing-support library to maven central eventually.